saleor/graphql/checkout/mutations.py

```diff
@@ -612,6 +613,13 @@ class Arguments:
                 "DEPRECATED: Will be removed in Saleor 4.0. Use token instead."
             ),
         )
+        customer_id = graphene.ID(
+            required=False,
+            description=(
+                "ID of customer to attach to checkout. Can be used to attach customer "
+                "to checkout by staff or app. Requires manage_users permission."
```

Comment:
In schema and in API reference permissions are capitalized enum values, so:
```suggestion
                "to checkout by staff or app. Requires MANAGE_USERS permission."
```